README.md

```
@@ -1,418 +1,331 @@
-```yaml
-apiVersion: v1
-kind: Pod
-metadata:
-  name: dmz-app
-  annotations:
-    k8s.v1.cni.cncf.io/networks: dmz-vlan81  # Attach to DMZ VLAN
-spec:
-  containers:
-  - name: app
-    image: nginx
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)